package-lock.json

```diff
@@ -1,28 +1,8038 @@
 {
   "name": "nueno",
+  "lockfileVersion": 2,
   "requires": true,
-  "lockfileVersion": 1,
```

Comment:
can you remove the package-lock file from this PR?

I think I already added a comment related to this. why did you resolve it?